src/fides/api/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -636,6 +638,78 @@ def get_request_status_logs(
     )
 
 
+@router.get(
+    PRIVACY_REQUEST_NOTIFICATIONS,
+    status_code=HTTP_200_OK,
+    response_model=PrivacyRequestNotificationInfo,
+    dependencies=[
+        Security(
+            verify_oauth_client,
+            scopes=[PRIVACY_REQUEST_NOTIFICATIONS_READ],
+        )
+    ],
+)
+def get_privacy_request_notification_info(
+    *, db: Session = Depends(deps.get_db)
+) -> PrivacyRequestNotificationInfo:
+    """Retrieve privacy request notification email addresses and number of failures to trigger notifications."""
+    info = PrivacyRequestNotifications.all(db)
+
+    if not info:
+        raise HTTPException(
+            status_code=HTTP_404_NOT_FOUND,
+            detail="No privacy request notification info found",
+        )
+
+    return PrivacyRequestNotificationInfo(
+        email_addresses=[x for x in info[0].email.split(", ")],
+        notify_after_failures=info[0].notify_after_failures,
+    )
+
+
+@router.put(
+    PRIVACY_REQUEST_NOTIFICATIONS,
+    status_code=HTTP_200_OK,
+    response_model=PrivacyRequestNotificationInfo,
+    dependencies=[
+        Security(
+            verify_oauth_client,
+            scopes=[PRIVACY_REQUEST_NOTIFICATIONS_CREATE_OR_UPDATE],
+        )
+    ],
+)
+def create_or_update_privacy_request_notifications(
+    *, db: Session = Depends(deps.get_db), request_body: PrivacyRequestNotificationInfo
+) -> PrivacyRequestNotificationInfo:
+    """Create or update list of email addresses and number of failures for privacy request notifications."""
+    if not request_body.email_addresses:
+        info = PrivacyRequestNotifications.all(db)
+        if info:
+            info[0].delete(db)
+        return PrivacyRequestNotificationInfo(
+            email_addresses=[],
+            notify_after_failures=request_body.notify_after_failures,
+        )
+
+    notification_info = {
+        "email": ", ".join(request_body.email_addresses),
```

Comment:
can we store `", "` as a const to be shared between here, the `GET` endpoint, and the `message_dispatch_service.py` (https://github.com/ethyca/fides/pull/1793/files?diff=unified&w=0#diff-a5422c23359e334382811df7a6ea20a2c00ecb53f52799038c4a9d7dd79fac7bR63)?